### PR TITLE
HW3

### PR DESCRIPTION
**Pull request by sempiternal21**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,6 +13,6 @@
 !settings.gradle
 !checkstyle.xml
 !.codeclimate.yml
-
+/src
```
Comment (Litemn) [inline .gitignore:16]: зачем?

**Comment by Litemn** [comment]
`ru.mail.polis.CompactionTest > overwrite(File) FAILED
    org.opentest4j.AssertionFailedError: expected: <true> but was: <false>
        at org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:39)
        at org.junit.jupiter.api.AssertTrue.assertTrue(AssertTrue.java:40)
        at org.junit.jupiter.api.AssertTrue.assertTrue(AssertTrue.java:35)
        at org.junit.jupiter.api.Assertions.assertTrue(Assertions.java:162)
        at ru.mail.polis.CompactionTest.overwrite(CompactionTest.java:80)
ru.mail.polis.CompactionTest > clear(File) FAILED
    org.opentest4j.AssertionFailedError: expected: <true> but was: <false>
        at org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:39)
        at org.junit.jupiter.api.AssertTrue.assertTrue(AssertTrue.java:40)
        at org.junit.jupiter.api.AssertTrue.assertTrue(AssertTrue.java:35)
        at org.junit.jupiter.api.Assertions.assertTrue(Assertions.java:162)
        at ru.mail.polis.CompactionTest.clear(CompactionTest.java:172)
ru.mail.polis.CompactionTest > multiple(File) FAILED
    org.opentest4j.AssertionFailedError: expected: <true> but was: <false>
        at org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:39)
        at org.junit.jupiter.api.AssertTrue.assertTrue(AssertTrue.java:40)
        at org.junit.jupiter.api.AssertTrue.assertTrue(AssertTrue.java:35)
        at org.junit.jupiter.api.Assertions.assertTrue(Assertions.java:162)
        at ru.mail.polis.CompactionTest.multiple(CompactionTest.java:122)`

падают тесты

File: .gitignore (hunk 1 above)
Comment (sempiternal21) [reply .gitignore]: Извиняюсь, привычка

**Comment by Litemn** [comment]
Принято.

**Pull request closed without merging**